go/paddlecloud/simplefile.go

```diff
@@ -87,8 +88,30 @@ func lsFile(dst string) error {
 		return errors.New("list file error: " + errMsg)
 	}
 	items := respObj.(map[string]interface{})["items"].([]interface{})
-	for _, i := range items {
-		fmt.Println(i.(string))
+	for _, fn := range items {
+		fmt.Println(fn.(string))
+	}
+	return nil
+}
+
+func putFiles(src string, dest string) error {
+	f, err := os.Stat(src)
+	if err != nil {
+		return err
+	}
+	switch mode := f.Mode(); {
+	case mode.IsDir():
+		files, err := ioutil.ReadDir(src)
+		if err != nil {
+			return err
+		}
+		for _, file := range files {
+			_, fn := path.Split(file.Name())
+			putFiles(path.Join(src, fn), path.Join(dest, fn))
```

Comment:
You mean the [filepath.Walk](https://golang.org/pkg/path/filepath/#Walk)?  It will list all the sub-files for the given directory, but I want to list the files hierarchy by hierarchy.